### DISCUSSION TITLE
Welcome to just Discussions!

### DISCUSSION THREAD

**Discussion by billywhizz** [just-js/just#30] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by viskogurl** [comment]
Hi, I'm Ian!

Planning on using Just to develop microservices and embedded systems involving prosthetics. I absolutely love Javascript and OSS so I'm looking forward to helping out with development!

As for what I do with this community, I haven't contributed anything besides some really bad puns... but just Await ;)

**Comment by tomByrer** [comment]
Hello!  Long-time programmer here.

Looking to use JustJS in production microservices / island front-ends SSR with [SolidJS](https://www.solidjs.com/)/React/Vue/[Mitosis](https://github.com/BuilderIO/mitosis).  Unsure of messaging system, likely will use [XState](https://xstate-catalogue.com/) state machines.  Also looking forward to move [my image comparitor](https://github.com/test-images/compare-image-files) into JustJS, but I need to ensure [sharp](https://github.com/lovell/sharp)/libvips runs here.

**Comment by noahehall** [comment]
planning to use nim + just as backends for various node apps
where node remains a backend-for-frontend, and nim/just are backends-for-backends

theres a general movement in the node community to integrate faster backends (bundling, transpiling etc) but the duality of nim + just (still working on that idea) seems to sync far better (syntax, paradigms, etc) than incorporating rust|go|etc

**Comment by criskurtin** [comment]
Hi! I'm Cristopher. A developer and an enthusiast :)

I'll be following this project closely, be it for personal or professional uses. It will certainly be considered for production usage in my future projects (might even port some from node) once it gets a bit more mature with the community and documentation.
I'll try to help a bit, however I can. This is certainly a chance to join a very interesting project in its early stages and possibly follow it through into a successful future.

Porting some of the commonly used libraries (npm packages) from node runtime to just seems like a great place to start with the expansion of ecosystem, so I guess I'll be doing a bit of that in my free time :)
